### PR TITLE
[14.0][MIG] l10n_be_vat_reports

### PR DESCRIPTION
**Pull request by huguesdk**

### CODE REVIEW

File: l10n_be_vat_reports/reports/be_vat_client_listing_consignment.xsd [hunk 1]
```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="http://www.minfin.fgov.be/ClientListingConsignment" xmlns:iso="http://www.minfin.fgov.be/IsoTypes" xmlns:common="http://www.minfin.fgov.be/InputCommon" targetNamespace="http://www.minfin.fgov.be/ClientListingConsignment" elementFormDefault="qualified" attributeFormDefault="unqualified" version="1.0" xml:lang="en">
+	<xs:import namespace="http://www.minfin.fgov.be/InputCommon" schemaLocation="IntervatInputCommon_v0_9.xsd"/>
+    <xs:import namespace="http://www.minfin.fgov.be/IsoTypes" schemaLocation="IntervatIsoTypes_v0_9.xsd"/>
```
Comment (remytms) [inline l10n_be_vat_reports/reports/be_vat_client_listing_consignment.xsd:4]: Here indentation can be made uniform.

File: l10n_be_vat_reports/reports/be_vat_intra_consignment.xsd [hunk 2]
```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="http://www.minfin.fgov.be/IntraConsignment" xmlns:iso="http://www.minfin.fgov.be/IsoTypes" xmlns:common="http://www.minfin.fgov.be/InputCommon" targetNamespace="http://www.minfin.fgov.be/IntraConsignment" elementFormDefault="qualified" attributeFormDefault="unqualified" version="1.0" xml:lang="en">
+	<xs:import namespace="http://www.minfin.fgov.be/InputCommon" schemaLocation="IntervatInputCommon_v0_9.xsd"/>
+    <xs:import namespace="http://www.minfin.fgov.be/IsoTypes" schemaLocation="IntervatIsoTypes_v0_9.xsd"/>
```
Comment (remytms) [inline l10n_be_vat_reports/reports/be_vat_intra_consignment.xsd:4]: Same here about indentation.

File: l10n_be_vat_reports/reports/be_vat_intra_consignment.xsd [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+					<xs:documentation xml:lang="de">Freier Kommentar</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+            <xs:element name="Justification" type="common:Justification_Type" minOccurs="0" maxOccurs="unbounded">
+                <xs:annotation>
+                    <xs:documentation xml:lang="fr">Information contenant une justification</xs:documentation>
+                    <xs:documentation xml:lang="nl">a faire</xs:documentation>
+                    <xs:documentation xml:lang="de">a faire</xs:documentation>
+                </xs:annotation>
+            </xs:element>
```
Comment (remytms) [inline l10n_be_vat_reports/reports/be_vat_intra_consignment.xsd:93]: This part is not indented as the rest of the file.

But also there is missing translation. Is that OK ?

File: l10n_be_vat_reports/reports/be_vat_intra_consignment.xsd [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+			</xs:annotation>
+		</xs:attribute>
+	</xs:complexType>
+	<xs:complexType name="IntraClientVATNumber_Type">
+		<xs:simpleContent>
+			<xs:extension base="common:EUVATNumber">
+				<xs:attribute name="issuedBy" type="iso:MSCountryCodeExclBE" use="required">
+					<xs:annotation>
+						<xs:documentation xml:lang="fr">Etat membre qui a attribu� le n� de TVA (autre que "BE")</xs:documentation>
+						<xs:documentation xml:lang="nl">Lidstaat die de btw-nummer heeft toegekend (autre que "BE")</xs:documentation>
```
Comment (remytms) [inline l10n_be_vat_reports/reports/be_vat_intra_consignment.xsd:183]: End of this line in NL needs to be translated: `(autre que "BE")`.

File: l10n_be_vat_reports/wizard/l10n_be_vat_intra.py [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+                        "MT",
+                        "NL",
+                        "PL",
+                        "PT",
+                        "RO",
+                        "SK",
+                        "SI",
+                        "ES",
+                        "SE",
+                        "GB",
```
Comment (remytms) [inline l10n_be_vat_reports/wizard/l10n_be_vat_intra.py:52]: GB still european country ? :p

**Review by remytms — COMMENTED** [review]
Maybe the big sql query can be a bit explained by comment. I read the code. But I didn't test the module.

**Comment by huguesdk** [comment]
@remytms thanks for the review!

xsd files are taken from the [intervat resources on the belgian government website](https://finances.belgium.be/fr/E-services/Intervat/documentation-technique) and are left unchanged (except converting crlf to lf), so no need to review their content.

i will comment the sql queries, it’s indeed needed, as it’s far from obvious.

about gb being still an european country, i didn’t change this, since i did not want to introduce a functional change as part of the migration.

**Review by remytms — APPROVED** [review]
LGTM

**Comment by vdewulf** [comment]
@huguesdk what are the next steps here to have this merged?
Thanks

**Comment by huguesdk** [comment]
we’re good regarding the code. now a psc will have to merge it. @robinkeunen maybe?

**Pull request merged**